src/JiraCli.Tests/IntegrationTests/IntegrationTests.cs

```diff
@@ -77,5 +79,20 @@ private Context CreateContext(string version)
             var createStableVersionContext = CreateContext("2.0.0");
             Assert.IsTrue(await action.Execute(createStableVersionContext));
         }
+
+        
+        [TestCase("TS-1,TS-2,TS-3")]
```

Comment:
Please add this test case as well:

[TestCase("TS-1, TS-2, TS-3")] 

It'll probably show that we should use Trim()